### ISSUE TITLE
conflict with vue/html-self-closing

### ISSUE THREAD

**Issue by aldarund** [prettier/eslint-config-prettier#85]
```
  extends: [
    '@nuxtjs',
    'plugin:nuxt/recommended',
    'plugin:prettier/recommended',
    'prettier',
    'prettier/vue',
  ],
  plugins: [
    'prettier'
  ],
```
It will lead to problem with for example img tags

Thats how prettier wants it
`<img src="~assets/img/lemken_icon.png" alt="L" />
`
Thats how vue eslint wants it
`
<img src="~assets/img/lemken_icon.png" alt="L">
`

eslint-config-prettier-check also dont report this rule as a problem

related prettier issue https://github.com/prettier/prettier/issues/5246

**Comment by lydell** [comment]
Hi!

`vue/html-self-closing` is already disabled:

https://github.com/prettier/eslint-config-prettier/blob/a2fceb9024838ea27c0c1b1184e7ff69ee76974f/vue.js#L5

Could you make a reproduction repo, where I can run `npm it` to reproduce?

**Comment by aldarund** [comment]
Hmmm, i cant reproduce it now... My only guess that there was some weird caching issue.. 
Closing for now

**Comment by cmygray** [comment]
I have the same issue. Every `<br>` tag is fixed to self-closing tag. 

https://gist.github.com/cmygray/f968f44c28832c1513679c688bd2fef0

**Comment by lydell** [comment]
@cmygray Prettier formats all void elements with a slash (`<br />`), yes. That's why the `vue/html-self-closing` rule conflicts/is redundant with Prettier. So eslint-config-prettier turns it off.

**Comment by qaptivator** [comment]
so the only way to fix this is to just disable the rule?
i have encountered the same issue while adding nuxt eslint module

**Comment by JounQin** [comment]
Yes, prettier is opinionated.
